debezium/debezium
# binh/incremental snapshot key

- wip
- Test deploy-file
- clean up
- Pass NOOP operations so that they will be visible downstream
- Pass raw_oplog into Debezium output (#3)
- wip (#4)
- Add debezium-bom (#5)
- Add debug logging for queue size (#6)
- Add debug logging in MongoDbStreamingChangeEventSource (#7)
- Revert "Add debug logging in MongoDbStreamingChangeEventSource (#7)" (#8)
- Add comment to oplog query (#10)
- Use bytes for raw oplogs (#11)
- Order imports (#12)
- Use DocumentCodec instead of BsonDocumentCodec (#13)
- Enforce RawBsonDocument for oplog capture mode (#14)
- Copy bytebuffer into arrary (#15)
- Use Bson value directly in JsonSerialization (#16)
- Remove extra check on enable oplog flag (#17)
- Allow find primary with secondary seed (#18)
- Support skip operation filter on oplog capture mode (#19)
- Allow skipping noop oplogs via skipped.operations config
- Support command op for mongo (#21)
- Add explicit safety check for command operation (#22)
- Fix getTransactionId
- DBZ-5003 Can't use 'local' database through mongos
- DBZ-5015 Get the json value of a Document when parsing signaling message
- DBZ-5015 Fix formatting
- DBZ-5011 Register MongoDB metrics with task id when more than one task
- DBZ-5260 Filter out unavailable replicaset members
- DBZ-5260 Replace Java 11 code
- Backport upstream changes from debezium-connector-mongodb v1.9.4.Final
- DBZ-5628 Added support for Mongo pre-image in change stream
- DBZ-5628 Add mongo 6.0 version and db version resolver
- DBZ-5628 Run CI with MongoDB 6.0
- DBZ-5628 Add mongo 6.0 pre-image IT test
- DBZ-5628 Fix formatting
- Fix cherry-pick
- Add raw BSON support
- fixup! Add raw BSON support
- fixup! fixup! Add raw BSON support
- Expose heartbeatFrequencyMs for mongo driver (#30)
- StripeAudit filtering on oplog mode (#33)
- Check if STRIPE_AUDIT exist first (#34)
- put toJson in debugMode
- Add metadata.yaml for fork repo (#36)
- Add wallTime in mongo source info (#37)
- Expose wallTime in parser (#38)
- DBZ-5973 Support for more types as key in mongodb incremental snapshot implementation
